### PR TITLE
[content update] 742-manipulate-data

### PR DESCRIPTION
**Pull request by DiSanchz**
This pull request contains the proposed updates for the building block [Manipulate Data](https://tilburgsciencehub.com/building-blocks/prepare-your-data-for-analysis/data-preparation/manipulate-data/), as part of the task assigned to Issue #742.

### Main comments on the non-updated version

- The building block is clear and the language appropriate
- Main link works
- No need to implement any suggestion by Grammarly
- **I'm inclined to think that the linked site has evolved since the building block's initial publication. Although it offers valuable machine learning and programming notes, it doesn't primarily focus on data manipulation. I suggest relocating the building block to the "Machine Learning" sub-section under "Analyze Data", and renaming it to "A collection of useful notes on Machine Learning" (Or something along those lines). Before making this shift, I wanted to consult with you, as I understand this wasn't part of the original update instructions. Conversely, if the original title and position within TSH's site are maintained and you approve the modifications described below, the Pull Request should be ready to merge. Let me know what you think about this.**

### Implemented modifications

- Keywords expanded
- Header image added to include some visual content to the building block. 
      - Note: _While the image is not related to the content of the Building block in a strict sense, I think that it works at making it somewhat more appealing visually._  
- Description deleted (metadata) as it was not adapted to the BBs content.

### CODE REVIEW

File: content/building-blocks/prepare-your-data-for-analysis/data-preparation/manipulate-data.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
 ---
 title: "Manipulate Data"
-description: "GitHub limits the size of files allowed in repositories. Use Git-LFS, an open-source Git extension for large file storage."
 weight: 3
-keywords: "manipulate, cleaning, wrangling, prep, data"
+keywords: "manipulate, cleaning, wrangling, prep, data, machone learning, visualization"
```
Comment (lachlandeer) [inline content/building-blocks/prepare-your-data-for-analysis/data-preparation/manipulate-data.md:4]: typo

**Comment by lachlandeer** [comment]
yes, this link has changed a lot

It looks like there's nothing much else in the BB. thoughts on whether its worth keeping @DiSanchz ?

File: content/building-blocks/prepare-your-data-for-analysis/data-preparation/manipulate-data.md (hunk 1 above)
Comment (DiSanchz) [reply content/building-blocks/prepare-your-data-for-analysis/data-preparation/manipulate-data.md]: fixed 👍

**Comment by DiSanchz** [comment]
> yes, this link has changed a lot
> 
> It looks like there's nothing much else in the BB. thoughts on whether its worth keeping @DiSanchz ?

If that is an option, given how much the link has changed indeed I would delete it @lachlandeer.

If you agree with that, I guess that I could just delete it from this branch before the PR is merged, right?

**Comment by hannesdatta** [comment]
This was indeed not very well aligned anymore. Thanks for identifying! I have deleted the reference.

**Pull request merged**